### DISCUSSION TITLE
helloWorld for ios/android

### DISCUSSION THREAD

**Discussion by jacobc94** [AvaloniaUI/Avalonia#15812] [Q&A] [answered]
Hello,
Can you please point me to a simple helloworld for ios and android as well as desktop? I am new to avalonia. I see the Avalonia.samples repo but those appear to be desktop only. Any help would be greatly appreciated.
Thank you very much.

**Comment by thevortexcloud** [comment, marked as answer]
Just run the built in cross platform template. It will generate a solution that will run on every platform Avalonia supports.

```
dotnet new avalonia.xplat
```
